load-testing/tests/relays_stress_test.go

```diff
@@ -349,7 +349,7 @@ func (s *relaysSuite) LocalnetIsRunning() {
 	s.setupTxEventListeners()
 
 	// Initialize the funding account.
-	s.initFundingAccount(fundingAccountAddress)
+	s.initFundingAccount(fundingAccountKeyName)
```

Comment:
@red-0ne I would push back on this. 

An address is the same across every environment and a single source of truth.

Anyone can import a private key with a different name.

IMO we should update all environments to use addresses instead of key names

Reply:
> Anyone can import a private key with a different name.

Anyone can also change the code to replace hard-coded addresses? Not sure what problem we're solving by using addresses in code. We can make address an environment variable as a workaround for this test.

Reply:
- keynames are subjective (can be different on different machines for the same address)
- addresses are objective (there is no room for discrepancy, confusion or cognitive overload)

Getting ahead of the conversations in the future

Reply:
The issue comes from the fact that different environments may have different funding addresses (pnf address was different in LocalNet and TestNet).

I agree that a keyname may reference different addresses, that in the future we'd even have to change it to `faucet` or similar and implicit changes to the keyname (e.g. regenesis, homedir change, config.yaml...) may even be harder to debug.

At the same time I understand @okdas 's concerns about changing the code just to run a TestNet load test.

While having this address in an environment variable might solve the issue, I would rather prefer having the manifest as a source of truth and introduce a `funding_account_address` entry and remove the hardcoded variable.